.github/dependabot.yml

```diff
@@ -42,13 +42,6 @@ updates:
     ignore:
       - dependency-name: "tanzu-framework"
 
-  - package-ecosystem: "gomod"
```

Comment:
This one shouldn't be removed as the `schedule` configuration is fine.

Reply:
Hi @stmcginnis did you check the failure of the dependabot pipeline? This line is duplicate of this https://github.com/vmware-tanzu/community-edition/blob/0a24ed9bace713bce0e8b6031da1a1009dd3b29f/.github/dependabot.yml#L10-L15

Reply:
The pipeline error I saw was that "schedule" could not be none, so I think the main issue is the one below. But yeah, line 10 already has this one, so +1 to removing it then.